packages/open-ui/src/main.tsx

```diff
@@ -1,3 +1,8 @@
-import { Button } from './components/Button';
-
-export { Button };
+export * from './components/AutoGrid';
```

Comment:
should we use named exports here @remotealex for naming consistency as mentioned in our WLDM experience ?

Reply:
These are named exported 👍